itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestMetaStoreLimitPartitionRequest.java

```diff
@@ -160,7 +160,7 @@ public void testMoreComplexQueryWithDirectSql() throws Exception {
   @Test
   public void testSimpleQueryWithDirectSqlTooManyPartitions() throws Exception {
     String queryString = "select value from %s where ds>'2008-04-20'";
-    executeQueryExceedPartitionLimit(queryString, 8);
+    executeQueryExceedPartitionLimit(queryString, 5);
```

Comment:
why do we change the tests?

Reply:
because we will try to get at most `limit + 1` (i.e 4 + 1 = 5) partitions after this patch.